doc/doxygen/src/board-cpu-features.md

```diff
@@ -0,0 +1,99 @@
+
+FEATURES                                                             {#features}
+========
+
+`FEATURES_PROVIDED` are hardware (including toolchain) features (e.g.:`periph_hwrng`,
```

Comment:
I somehow precise it with `BSP board support package` to include also build system, build system configuration, flasher/tooling and not only `toolchain` that could be misinterpreted as `gcc/ld` only.